set.mm

```diff
@@ -416797,6 +416901,36 @@ sequence p(0) p(1) ... p(n) of the vertices in a walk p(0) e(f(1)) p(1)
       WKWAMTKZDVTWAMTUPVQDURIWPDODUSDUTVAVBVCVDVEVFVGVIVJVHVKVLVMVNVO $.
   $}
 
+  ${
+    $d G i $.  $d N i $.  $d W i $.
+    $( Other basic properties of a walk of a fixed length as word.
+       (Contributed by AV, 8-Mar-2022.) $)
+    wwlknbp1 $p |- ( W e. ( N WWalksN G )
+         -> ( N e. NN0 /\ W e. Word ( Vtx ` G ) /\ ( # ` W ) = ( N + 1 ) ) ) $=
+      ( vi cwwlksn co wcel cvv cn0 cvtx cfv cword w3a chash caddc wceq cpr eqid
+      c1 cv cedg cc0 cfzo wral wwlknbp wwlknp wi wa simpl simpr1 simpr2 3jca ex
+      3ad2ant2 sylc ) CBAEFGAHGZBIGZCAJKZLGZMUSCNKBSOFPZDTZCKVASOFCKQAUAKZGDUBB
+      UCFUDZMZUQUSUTMZABURCURRZUEDVBABURCVFVBRUFUQUPVDVEUGUSUQVDVEUQVDUHUQUSUTU
+      QVDUIUQUSUTVCUJUQUSUTVCUKULUMUNUO $.
+  $}
+
+  $( Yet other basic properties of a set being a walk of length n as word.
+     (Contributed by Alexander van der Vekens, 3-Oct-2018.)  (Revised by AV,
+     12-Apr-2021.)  (Proof shortened by AV, 8-Mar-2022.) $)
+  wwlknbp2 $p |- ( W e. ( N WWalksN G )
+                     -> ( W e. Word ( Vtx ` G ) /\ ( # ` W ) = ( N + 1 ) ) ) $=
+    ( cwwlksn co wcel cn0 cvtx cfv cword chash c1 caddc wceq wa wwlknbp1 3simpc
+    w3a syl ) CBADEFBGFZCAHIJFZCKIBLMENZRUAUBOABCPTUAUBQS $.
```

Comment:
This theorem does not seem to bring in much compared to the previous one.

Reply:
Yes, you are right, but ~wwlknbp2 is the older theorem which is used 11 times. I have to check each of these 11 theorems if their proofs can be shortened by using ~wwlknbp1, or at least do not become longer. If this is the csase, I will remove ~wwlknbp2. I put it on my TODO list.